projects/canopy/src/lib/side-nav/side-nav.component.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../styles/mixins';
+
+.lg-side-nav {
+  display: flex;
+  flex-direction: column-reverse;
+
+  @include lg-breakpoint('md') {
```

Comment:
You don't need this I don't think, as you have already set that for all breakpoints above

Reply:
I've removed it.